lib/runner/variant-reassignment.js

```diff
@@ -157,14 +157,14 @@ export default class VariantReassignment {
   async _processUnfinishedTransactions (transactions, retryCount = 0) {
     try {
       if (!transactions) {
-        log('Loading unfinished transactions')
+        this.logger.info('Loading unfinished transactions')
         transactions = await this.transactionService.getTransactions()
       }
 
       for (const transactionObject of transactions) {
         const { key, value: transaction } = transactionObject
 
-        this.logger.debug(`Processing unfinished transaction with key ${key}`)
+        this.logger.info(`Processing unfinished transaction with key ${key}`)
```

Comment:
Shouldn't we log as minimum as possible?

Reply:
I agree that in the normal case we should not log unnecessary stuff. But when things go wrong, it can get handy to have more info so we know what exactly happened there.

Reply:
OK let's leave it for now, if it's too much, later we can change it.